### DISCUSSION TITLE
Is it possible / supported to set up IAS warn zone directly between Heiman Smoke Sensors?

### DISCUSSION THREAD

**Discussion by fribse** [Koenkk/zigbee2mqtt#14139] [Devices]
I have some of the Heiman [HS1SA](https://www.zigbee2mqtt.io/devices/HS1SA.html#heiman-hs1sa).
They are of course connected to my z2m radio, and I've made flows in Node Red for alerting on sirens, cellphones etc.
It would be great to be able have the smoke sensors alert each other directly, so that even if z2m and HA fails, they would still start each other up.

**Comment by Koenkk** [comment]
AFAIK the Zigbee spec does not define such functionallity.

**Comment by fribse** [comment]
Darned, well, that's what I thought ;-)
